config/routes.rb

```diff
@@ -13,5 +13,15 @@
       end
       resources :comments
     end
-  end  
+  end
+
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :posts do
```

Comment:
- [Optional] Dear Team, for the `users` and `posts` I can see you have defined a resource, but creating a resource will add a lot of routes that we might not need. I would suggest using `only` whenever we create a resource and will not use all the CRUD operations on it.

Reply:
Thank you very much @TadesseDev , I addressed your optional change 🥳